### DISCUSSION TITLE
Take values in application deployment manifest file from Values-test.yaml file

### DISCUSSION THREAD

**Discussion by rishindra-singh** [argoproj/argo-cd#9341] [Q&A]
Hello Everyone,

I am trying to deploy an application on ArgoCD using CLI from the manifest file. In manifest file I have to pass some values like destination server, destination namespace, etc.

I want Argo to read this values from values-test.yaml file while deploying and deploy my application. I don't want to pass this values in manifest file.

**Comment by rishabh625** [comment]
I would suggest you to use applications set for this git file generator can help u with this
https://argocd-applicationset.readthedocs.io/en/stable/Generators-Git/#git-generator-files
